Resources/config/imagine.xml

```diff
@@ -32,6 +32,39 @@
 
     <services>
 
+        <!-- Factory services -->
+        <service id="liip_imagine.factory.config.filter_set" class="Liip\ImagineBundle\Factory\Config\FilterSetFactory" />
+        <service id="Liip\ImagineBundle\Factory\Config\FilterSetFactory" alias="liip_imagine.factory.config.filter_set"/>
+
+        <service id="liip_imagine.factory.config.filter.crop" class="Liip\ImagineBundle\Factory\Config\Filter\CropFactory" />
+        <service id="Liip\ImagineBundle\Factory\Config\Filter\CropFactory" alias="liip_imagine.factory.config.filter.crop"/>
+
+        <service id="liip_imagine.factory.config.filter.strip" class="Liip\ImagineBundle\Factory\Config\Filter\StripFactory" />
+        <service id="Liip\ImagineBundle\Factory\Config\Filter\StripFactory" alias="liip_imagine.factory.config.filter.strip"/>
+
+        <service id="liip_imagine.factory.config.filter.thumbnail" class="Liip\ImagineBundle\Factory\Config\Filter\ThumbnailFactory" />
+        <service id="Liip\ImagineBundle\Factory\Config\Filter\ThumbnailFactory" alias="liip_imagine.factory.config.filter.thumbnail"/>
```

Comment:
i don't think that we need an autowiring capable alias for these. they should be private services not useable outside of DI configuration

Reply:
done